src/applications/disability-benefits/all-claims/tests/all-claims-revised-add-disabilities.cypress.spec.js

```diff
@@ -0,0 +1,40 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import { cloneDeep } from 'lodash';
+import mockUser from './fixtures/mocks/user.json';
+import mockFeatureToggles from './fixtures/mocks/feature-toggles.json';
+import formConfig from '../config/form';
+import manifest from '../manifest.json';
+import { setup, pageHooks } from './cypress.helpers';
+
+const toggles = cloneDeep(mockFeatureToggles);
+toggles.data.features.push({
+  name: 'disability_526_improved_autosuggestions_add_disabilities_page',
+  value: true,
+});
+
+const testConfig = createTestConfig(
+  {
+    dataPrefix: 'data',
+    useWebComponentFields: true,
+
+    dataSets: ['newOnly-test'],
+
+    fixtures: {
+      data: path.join(__dirname, 'fixtures', 'data'),
+    },
+
+    pageHooks: pageHooks(cy, mockFeatureToggles),
+    setupPerTest: () => {
+      cy.login(mockUser);
+      setup(cy, mockFeatureToggles);
```

Comment:
```suggestion
      setup(cy, toggles);
```

The test runs successfully, but we need these changes to enable the revised page